website/docs/r/automation_runbook.html.markdown

```diff
@@ -106,7 +106,11 @@ The following arguments are supported:
 
 * `log_activity_trace_level` - (Optional) Specifies the activity-level tracing options of the runbook, available only for Graphical runbooks. Possible values are `0` for None, `9` for Basic, and `15` for Detailed. Must turn on Verbose logging in order to see the tracing.
 
-* `draft` - (Optional) A `draft` block as defined below .
+* `draft` - (Optional) A `draft` block as defined below.
+
+* `job_schedule` - (Optional) Onr or more `job_schedule` block as defined below.
+
+~> **NOTE** AzureRM provides a stand-alone [azurerm_automation_job_schedule](automation_job_schedule.html.markdown) and this inlined `job_schdule` property to manage the job schedules. At this time you should choose one of them to manage the job schedule resources.
```

Comment:
Can we also put this notice at the begining of the `automation_job_schedule` document?